### PR TITLE
add checks for docker rss and vms

### PR DESCRIPTION
**Pull request by BlueShells**
this script will collect the docker memory usage rss and vms ,but not on the master (if we need the usage of docker master ,I will change the function ) and the application is "Docker Daemon"

### CODE REVIEW

File: scripts/monitoring/cron-send-docker-metrics.py [hunk 1]
```diff
@@ -39,6 +60,13 @@
             'docker.storage.is_loopback': int(du_dds.is_loopback),
             'docker.ping': 1, # Docker is up
         }
+        if docker_memoryusage.has_key('rss'):
+            keys['docker.memoryuage.rss'] = int(docker_memoryusage['rss'])
```
Comment (drewandersonnz) [inline scripts/monitoring/cron-send-docker-metrics.py]: should this be: memoryusage with an 's'?

File: scripts/monitoring/cron-send-docker-metrics.py [hunk 2]
```diff
@@ -39,6 +60,13 @@
...
+            keys['docker.memoryuage.vms'] = int(docker_memoryusage['vms'])
```
Comment (drewandersonnz) [inline scripts/monitoring/cron-send-docker-metrics.py]: should this be: memoryusage with an 's'?

File: ansible/roles/os_zabbix/vars/template_docker.yml [hunk 3]
```diff
@@ -17,6 +17,16 @@ g_template_docker:
     - Docker Daemon
     value_type: int
 
+  - key: docker.memoryuage.rss
+    applications:
+    - Docker Daemon
+    value_type: int
+
+  - key: docker.memoryuage.vms
```
Comment (drewandersonnz) [inline ansible/roles/os_zabbix/vars/template_docker.yml]: should this be: memoryusage with an 's'?

File: ansible/roles/os_zabbix/vars/template_docker.yml [hunk 4]
```diff
@@ -17,6 +17,16 @@ g_template_docker:
     - Docker Daemon
     value_type: int
 
+  - key: docker.memoryuage.rss
```
Comment (drewandersonnz) [inline ansible/roles/os_zabbix/vars/template_docker.yml]: should this be: memoryusage with an 's'?

**Review by drewandersonnz — CHANGES_REQUESTED** [review]
should this be: memoryusage with an 's'?

**Comment by twiest** [comment]
[test]

**Comment by twiest** [comment]
[test]

**Comment by twiest** [comment]
[test]

**Comment by twiest** [comment]
@BlueShells The bot is working now. The failure above is a valid failure (you need to fix some stuff).

See the output of the build bot for more details.

**Review by drewandersonnz — APPROVED** [review]
+1

**Pull request merged**